### ISSUE TITLE
Some links for the members in involvements are not active

### ISSUE THREAD

**Issue by charlielee144r** [gordon-cs/gordon-360-ui#1748]
Some links for members in an involvements are not active, while some are

**Comment by charlielee144r** [comment]
Referenced in #1736

**Comment by bennettforkner** [comment]
Is this not a duplicate of #1638 which you closed as completed @charlielee144r ? Also I'm not seeing where you reference this issue in #1736.

If it's not a duplicate, can you give a little more qualifying information as to what distinguishes this issue from the other (or what you fixed to resolve the other PR)?

FYI, good practice is to use the issue's '#' code on the actual PR itself. It will then make a note here that you mentioned it there.

**Comment by charlielee144r** [comment]
@bennettforkner I made a new issue because I didn't think it was really related to the issue I closed. PR #1736 was specifically about fixing a variable to validate links on smaller screens (when pressing the link on smaller screens, it would just redirect to an invalid page). I made this a new issue because it was related to member links not even showing up in involvements for certain members. 

I will make sure to mention the issue's # on the PRs next time!

This issue was resolved by correcting logic in the database.
